packages/config/src/types.ts

```diff
@@ -37,28 +81,90 @@ export type Config = {
     unsubscribe: () => void;
   };
 
+  /**
+   * Checks whether the given key is present.
+   */
   has(key: string): boolean;
 
+  /**
+   * Lists all available configuration keys.
+   */
   keys(): string[];
 
+  /**
+   * Same as `getOptional`, but will throw an error if there's no value for the given key.
+   */
   get<T = JsonValue>(key?: string): T;
+
+  /**
+   * Read out all configuration data for the given key.
+   *
+   * Usage of this method should be avoided as using the typed alternatives provide
```

Comment:
```suggestion
   * Usage of this method should be avoided as using the typed alternatives provides
```

Reply:
But there are so many alternatives

Reply:
But only one using. (It really should be singular :))

Reply:
Wat, no

Reply:
@timbonicus

Reply:
The count of things doesn't come into play at all. Try putting "leads to" there instead to get a feel for it.

"Using X provides better Y"

It's the usage that provides, not the X.

Reply:
Repetition and discussion gone x)

Reply:
Edit LGTM!